cdk.json

```diff
@@ -40,6 +40,7 @@
         "OwnerEmail": "synapseeng@sagebase.org"
       },
       "STACK_NAME_PREFIX": "registry-dev",
+      "BUCKET_NAME": "dev-docker-registry.sagebase.org",
```

Comment:
bucket names are unique in a region.  To ensure a bucket name is unique i would not set this param and let AWS create a unique bucket name.  If you insist on naming the bucket then I would add add the AWS account ID somewhere to the name.

Reply:
> bucket names are unique in a region. 

In fact bucket names are unique across all standard regions, not just within a region.  From [the doc's](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingBucket.html)

> Amazon S3 supports global buckets, which means that each bucket name must be unique across all AWS accounts in all the AWS Regions within a partition. A partition is a grouping of Regions. AWS currently has three partitions: aws (Standard Regions), aws-cn (China Regions), and aws-us-gov (AWS GovCloud (US)).

I like the idea of letting CloudFormation create a unique name for a bucket however the Docker registry already has a bucket containing production data, and this deployment will have to link to that existing bucket, so CF can't choose the bucket name.

Reply:
ahh ok, i thought this was creating the buckets.